documentation/modules/exploit/windows/local/lexmark_driver_privesc.md

```diff
@@ -0,0 +1,108 @@
+## Vulnerable Application
+
+Various Lexmark Universal Printer drivers as listed at [advisory TE953](http://support.lexmark.com/index?page=content&id=TE953)
+allow low-privileged authenicated users to elevate their privileges to `SYSTEM` on affected Windows systems by modifying
+the XML file at `C:\\ProgramData\\<driver name>\\Universal Color Laser.gdl` to replace the DLL path
+to `LMUD1OUE.DLL` with a malicious DLL path.
```

Comment:
This is going to need to be updated to `unires.dll` now that we changed the name of the file we are replacing.